### DISCUSSION TITLE
why changing direkt links on wiki pages to linkify.me

### DISCUSSION THREAD

**Discussion by k3b** [k3b/APhotoManager#194] [Q&A]
@ma-r-vin  hello marvin. Out of curiosity: can you tell me why you changed the direct links from the "a photo mananger" wikipages to linkyfy-me-links ?

i.e. you changed from https://a-photo-manager.en.uptodown.com/android to https://linkify.me/0N2sQXH

is there any benefit doing it?

you can reply me her by adding @k3b .

If i get no reply within 7 days i will revert your changes.

**Comment by vinceh121** [comment]
To save you a few clicks messing around, this is the UI that linkify.me provides.
![](https://i.vinceh121.me/pbuiRk6e.png)
![](https://i.vinceh121.me/XoLDeQke.png)

**Comment by k3b** [comment]
thakns for the rply. since i donot like user-tracking i have removed the modified links
